AGS.gitignore

```diff
@@ -0,0 +1,34 @@
+# Adventure Game Studio git ignore
+# --------------------------------
+
+# Built things
+_Debug/
+Compiled/
+
+# AudioCache can be rebuilt from sources
+AudioCache/
+
+# Lockfile
+_OpenInEditor.lock
+
+# User settings
+Game.agf.user
+*.crm.user
+
+# Backups
+Game.agf.bak
+backup_acsprset.spr
+
+# Memory dumps
+*.dmp
+
+# Temporary files
+# temporarily created during sprite or room background compression
+~aclzw.tmp
+# temporary, main game data, before getting packed into exe
+game28.dta
+# temporary build of the game before being moved to Compiled/ folder
+*.exe
```

Comment:
Ignoring *.exe files may be a little extreme. Is there anyway to figure out the temporary name of this file?

Reply:
It will be equal to game's name.
I don't think it's okay to commit exe files inside AGS game project anyway. But if you believe it should be avoided, then perhaps remove it. Normally this file appears in the game project folder for mere seconds during compilation.